Apps/Database/src/Delegates/DBCommunicationDelegate.cs

```diff
@@ -93,5 +93,17 @@ public DBResult<Communication> Add(Communication communication, bool commit = tr
             this.logger.LogDebug($"Finished adding Communication in DB");
             return result;
         }
+
+        /// <inheritdoc />
+        public DBResult<IEnumerable<Communication>> GetList()
+        {
+            this.logger.LogTrace($"Getting all communication entries...");
+            DBResult<IEnumerable<Communication>> result = new DBResult<IEnumerable<Communication>>();
+            result.Payload = this.dbContext.Communication
+                    .OrderBy(o => o.CreatedDateTime)
+                    .ToList();
+            result.Status = result.Payload != null ? DBStatusCode.Read : DBStatusCode.NotFound;
```

Comment:
Can you confirm that the if the table is empty the payload is *empty* and not *null*? if it gets set to "not found" the service will return the request result as error which i think its not right since the list could be empty.